docs/run/sessions.mdx

```diff
@@ -134,13 +134,7 @@ If the maximum timeout value has not been reached:
     -   No further jobs can be submitted to the session.
     -   The session cannot be reopened.
 
-## Different ways of using sessions
-
-Sessions can be used for iterative or batch execution.
-
-### Iterative
-
-Any session job submitted within the interactive timeout, also known as interactive time to live (ITTL), is processed immediately. This allows some time for variational algorithms, such as VQE, to perform classical post-processing.
+Sessions run _iteratively_.  To run jobs in batch mode, see [batch](run-primitives-batch).   Running iteratively means that any session job submitted within the interactive timeout, also known as interactive time to live (ITTL), is processed immediately. This allows some time for variational algorithms, such as VQE, to perform classical post-processing.
```

Comment:
```suggestion
Sessions run _iteratively_. Running iteratively means that any session job submitted within the interactive timeout, also known as interactive time to live (ITTL), is processed immediately. This allows some time for variational algorithms, such as VQE, to perform classical post-processing. To run jobs in batch mode, see [batch](run-primitives-batch).
```
It feels a little odd to introduce the idea of iteration then jump to batching then go back to explaining iteration, do i'd recommend rephrase like this